arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "eventHubEndpointName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "name",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/Object"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/providers/Microsoft.Devices/operations": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all supported Resource Provider Operations",
+				"description": "Get all supported Resource Provider Operations",
+				"operationId": "IotHubResource_GetResourceProviderOperations",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResponseWithContinuation[IEnumerable[ResourceProviderOperationInfo]]"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Devices/IotHubs": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all IotHubs in a subscription",
+				"description": "Get all IotHubs in a subscription",
+				"operationId": "IotHubResource_GetIotHubsForSubscription",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResponseWithContinuation[IEnumerable[IotHubDescription]]"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all IotHubs in a Resourcegroup",
+				"description": "Get all IotHubs in a Resourcegroup",
+				"operationId": "IotHubResource_GetIotHubsForResourceGroup",
```

Comment:
Expoected format: IotHubResource_ListByResourceGroup. This will bring consistency.
#244 documents this issue and we are working towards bringing consistency for standard operations across all the RPs so that the customer gets the same experience.